### PR TITLE
Add Profile RayyanRNKJ

### PR DESCRIPTION
**Pull request by RayyanRNKJ**

### CODE REVIEW

File: public/data/RayyanRNKJ.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      "url": "https://github.com/RayyanRNKJ",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/RayyanRNKJ",
+      "icon": "twitter"
+    }
+  ]
+}
```
Comment (krishguptadev) [inline public/data/RayyanRNKJ.json:17]:
```suggestion
}

```

File: public/data/RayyanRNKJ.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Rayyan Rasheed",
+  "bio": "👨‍💻 React JS Developer | UI Designer | Freelancer | Self Learner | 14-Year-Old",
+  "avatar": "https://github.com/RayyanRNKJ.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
```
Comment (krishguptadev) [inline public/data/RayyanRNKJ.json:7]:
```suggestion
      "name": "Check out my GitHub",
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Good work, but a few *suggestions* from my side 😄

**Review by schmelto — APPROVED** [review]
image

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
**Thanks for your contribution** 🚀
@krishguptadev thanks for reviewing

**Pull request merged**